pda.sh

```diff
@@ -0,0 +1,42 @@
+package: PDA
+version: "%(tag_basename)s"
+tag: 11.0.7
+source: https://github.com/PascalBoeschoten/pda
```

Comment:
We can either for one of the two above (whichever you prefer) or have our own standalone `AliceO2Group/pda` where we sync the changes by hand. Whatever is more convenient to you. Once you decide let me know and I can create the repo and give you push rights.